.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": ["eslint:recommended"],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
I think these rules that you set here are what is causing you to fail the stickler tests.

What were the options we selected when we installed `eslint` on this project (i.e, thru `npx eslint --init`)? We should have:

- Selected AirBnB style guide
- Selected `Browser` as to where we are running our files
- Selected `import/export` statements when asked